### DISCUSSION TITLE
OCR code and support

### DISCUSSION THREAD

**Discussion by waywardmonkeys** [Interlisp/medley#126] [Implementation history]
Is there any information about what the OCR code in Maiko was for? Where was it used? Did it work? Does anything important rely on it?

**Comment by masinter** [comment]
I think OCR and the linear programming package were motivations for Foreign Function call. Interfaces to obsolete software packages we'd do another way if we had to.  That and keymaker and vdate.c.

**Reply by nbriggs** [reply]
vdate.c is useful -- it give us the creation date of the executable with `lde -info`
